### PR TITLE
DEVX-1593 updates broken links in README to demo-scene

### PR DESCRIPTION
**Pull request by rspurgeon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,8 +43,9 @@ The best demo to start with is [cp-demo](https://github.com/confluentinc/cp-demo
...
-| [KSQL workshop](https://github.com/confluentinc/demo-scene/blob/5.4.x/ksql-workshop/)   |   N   |   [Y](https://github.com/confluentinc/demo-scene/blob/5.4.x/ksql-workshop/)    | showcases Kafka event stream processing using KSQL and can run self-guided as a KSQL workshop <br><img src="https://github.com/confluentinc/demo-scene/blob/5.4.x/ksql-workshop/images/ksql_workshop_01.png" width="450">
+| [KSQL UDF](https://github.com/confluentinc/demo-scene/blob/master/ksql-udf-advanced-example/README.md?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.top) | [Y](https://github.com/confluentinc/demo-scene/blob/master/ksql-udf-advanced-example/README.md?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.top) | N | Advanced [KSQL User-Defined Function (UDF)](https://www.confluent.io/blog/build-udf-udaf-ksql-5-0?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.top) use case for connected cars <br><img src="https://www.confluent.io/wp-content/uploads/KSQL-1-350x195.png" width="350">
+| [KSQL workshop](https://github.com/confluentinc/demo-scene/blob/master
```
Comment (ybyzek) [inline README.md]: Link is broken due to newline

**Review by ybyzek — CHANGES_REQUESTED** [review]
@rspurgeon thanks for the quick PR.  Small required fix before merge

File: README.md (hunk 1 above)
Comment (rspurgeon) [reply README.md]: grr, not sure how I missed that... updated

**Review by ybyzek — APPROVED** [review]
LGTM

**Pull request merged**